inventory/actions/service/bulk-check-expirations.php

```
@@ -0,0 +1,34 @@
+<?php
+/*
+    This file is part of Symbiose Community Edition <https://github.com/yesbabylon/symbiose>
+    Some Rights Reserved, Yesbabylon SRL, 2020-2024
+    Licensed under GNU AGPL 3 license <http://www.gnu.org/licenses/>
+*/
+
+
+// announce script and fetch parameters values
+list($params, $providers) = announce([
+    'description'	=>	"Bulk verify subscription expiration.",
+    'params' 		=>	[
+        'ids' => [
+            'description'       => 'List of Subscription identifiers the check against emptyness.',
+            'type'              => 'array'
+        ]
+    ],
+    'response' => [
+        'content-type'      => 'application/json',
+        'charset'           => 'utf-8',
+        'accept-origin'     => '*'
+    ],
+    'providers' => ['context']
+]);
+
+list($context) = [$providers['context']];
+
```

Maybe throw invalid param error if $params['ids'] is empty